PF/PF-Assignment1/Cart.java

```diff
@@ -0,0 +1,62 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+
+class Cart{
+	
+	private double totalPrice;
+	private int totalItems;
+	private ArrayList<String> list;
+	HashMap<Integer, Double> itemPriceList; 
```

Comment:
Try to add access specifiers to all the class members and methods according to best practices, always keep members (variables) as private and access them using getters and setters.